documentation/asciidoc/computers/remote-access/find-your-ip-address.adoc

```diff
@@ -89,16 +95,16 @@ Once you've identified the correct network interface block, look for a field nam
 
 In the example above, the Raspberry Pi uses Wi-Fi to access the internet. Check the block where the `GENERAL.TYPE` field reads "wifi" to find the IP address. In this case, you can access this device using the IPv4 address in the `IP4.ADDRESS[1]` field: `192.168.1.42`.
 
-==== Resolving `raspberrypi.local` with mDNS
+=== Resolve `raspberrypi.local` with mDNS
 
 On Raspberry Pi OS, multicast DNS is supported out-of-the-box by the Avahi service.
 
 If your device supports mDNS, you can reach your Raspberry Pi by using its hostname and the `.local` suffix.
 The default hostname on a fresh Raspberry Pi OS install is `raspberrypi`, so by default any Raspberry Pi running Raspberry Pi OS responds to:
```

Comment:
Might be worth adding a link here to the OS Customisation section of the RPi Imager instructions, as that also allows you to change your hostname. I suspect this documentation predates Raspberry Pi Imager.